File: chartiq/src/main/java/com/chartiq/sdk/model/DrawingParameter.java
package com.chartiq.sdk.model;

public enum DrawingParameter {
    FILL_COLOR("fillColor"),
    CURRENT_COLOR("currentColor"),
    PATTERN("pattern"),
    LINE_WIDTH("lineWidth");

    String value;

    DrawingParameter(String value) {
        this.value = value;
    }
}

File: chartiq/src/main/java/com/chartiq/sdk/model/drawingtool/DrawingParameter.kt
package com.chartiq.sdk.model.drawingtool

// TODO: 20.11.20 Discuss if it's possible to keep the DrawingParameter for Typed requests
enum class DrawingParameter(val value: String) {
    LINE_TYPE("pattern"),
    LINE_WIDTH("lineWidth"),
    COLOR("color"),
    FILL_COLOR("fillColor")
}
